Update communities.md

wittigue opened the pull request:
new text about FAIRDOM Community Engagement

wittigue committed:
Update communities.md

new text about FAIRDOM Community Engagement

PhilReedData commented on pages/communities/communities.md:
```diff
@@ -1,43 +1,17 @@
 
-* [Digital Salmon](https://fairdomhub.org/projects/38)
-* [Plant Systems-biology Modelling](https://fairdomhub.org/projects/129)
+[Biohackathon](https://biohackathon-europe.org) is the hackathon run by ELIXIR, the European ESFRI Infrastructure for Bioinformatics. The website also comprises a list of projects that give an impression. FAIRDOM typically runs or contributes projects in these hackathons. [de.NBI](https://www.denbi.de), the organisation behind the German node of ELIXIR runs a similar set of hackathons (https://ww ...
```
I like to include BioHackathon here, but what is meant by "impression" here? Do you mean, the list of projects gives a suggestion of what kinds of projects FAIRDOM is a part of at past BioHackathon events?

wittigue replied:
May be we should rephrase it. FAIRDOM is involved in several projects but I would not list them here separately.

PhilReedData replied:
`[Biohackathon](https://biohackathon-europe.org) is the hackathon run by ELIXIR, the European ESFRI Infrastructure for Bioinformatics. [de.NBI](https://www.denbi.de), the organisation behind the German node of ELIXIR runs a similar set of hackathons (https://www.denbi.de/de-nbi-events). FAIRDOM is typically involved in several projects of these hackathons each year.`

PhilReedData commented on pages/communities/communities.md:
```diff
@@ -1,43 +1,17 @@
-* [MESI-STRAT](https://fairdomhub.org/projects/99)
-* [COVID-19 Disease Maps](https://fairdomhub.org/projects/190)
-* [PoLiMeR](https://fairdomhub.org/projects/130)
+From 2012 to 2018 we ran four so-called foundry meetings with practitioners in support delivery for systems biology all over Europe, to exchange ideas and practices, to share software and to interoperate systems where useful to do so. We did not continue these meetings as we feel the grounds of the foundry meetings is covered by two sets of activities, the Systems Biology Community in ELIXIR that ...
```
I will tweak this slightly. 

- I'll replace 'so-called' with inverted commas around 'foundry'. 
- I'll change the wording for why we did not continue from 'feel' to 'found', to sound more evidence-based.

stuzart commented on pages/communities/communities.md:
```diff
@@ -1,43 +1,17 @@
-
-### Other Domains
-
-* HITS-Publications
```
could the HITS publication instance be included in FAIRDOM in Use ? I think it is a nice example of a particular use-case, and has contributed enhancements

PhilReedData replied:
Good idea. Do you know who to approach for HITS publications, please?

wittigue replied:
@whomingbird Xiaoming is working on HITS publications SEEK. We could write a short description for the website together.

PhilReedData commented on pages/communities/communities.md:
```diff
@@ -1,43 +1,17 @@
-* [COVID-19 Disease Maps](https://fairdomhub.org/projects/190)
-* [PoLiMeR](https://fairdomhub.org/projects/130)
+From 2012 to 2018 we ran four so-called foundry meetings with practitioners in support delivery for systems biology all over Europe, to exchange ideas and practices, to share software and to interoperate systems where useful to do so. We did not continue these meetings as we feel the grounds of the foundry meetings is covered by two sets of activities, the Systems Biology Community in ELIXIR that ...
 
```
In addition...

Suggest we summarise the PALs page history with a paragraph here, instead of reinstating the PALs page changed to the past tense. That way, we don't highlight/advertise PALs too much (ie keep it out of the site navigation menu). 

This will need adjusting, but something like...

`From 2011 to 2022, FAIRDOM PALs were “front line” experimentalists, modellers and bioinformaticians from each of the projects we work with. They acted as our advocates and communicate our progress back to their projects. Today, we encourage advocacy through our dedicated [community channels and regular meetings](/get_involved). For more about PALs, see the [PALs presentations](https://fairdomhub.org/presentations?filter%5Bquery%5D=pals&order=created_at_desc). `

wittigue replied:
Great idea, thanks!

PhilReedData commented:
Thanks @wittigue, I will implement these minor changes into the code then merge the branch.

PhilReedData committed:
Update communities.md

Now that the FAIRDOM In Use section is live, make the Communities page better cover other aspects of FAIRDOM community involvement. Capture the retired PALs content here too.

PhilReedData committed:
Merge pull request #96 from FAIRdom/master

Bring communities branch up to date before completion

PhilReedData committed:
Tested new community engagement page
